rosdep/python.yaml

```diff
@@ -4170,6 +4170,13 @@ python-responses-pip:
   ubuntu:
     pip:
       packages: [responses]
+python-rhino3dm:
+  ubuntu: 
+    pip:
+      packages: [rhino3dm]
+  focal: 
+    pip:
+      packages: [rhino3dm]
```

Comment:
I'm not sure what are you trying to do, but `focal` is an ubuntu version, cannot be added as a distro.

```suggestion
```